experience.tex

```diff
@@ -6,33 +6,35 @@ \section{EXPERIENCE}
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Worked with the Supply Chain team to determine requirements and metric thresholds for displaying data to
-        the service providers based on their performance for previously completed work items.
+        Assisted building and maintaing greenfield iOS application written in SwiftUI to replace legacy Xamarin application,
+        reducing crash rate from 9\% to < 1\%, reducing time for local development and time to ship a new version of the app,
+        and added thorough logging to the application to make errors actionable for development team.
     }
     \item{
-        Utilized AWS Elasticsearch to improve filtering for assets by properties and to quickly return large lists of
-        matching data to the primary asset web service, lowering asset search response times for the end user.
+        Containerized legacy C\# applications to run in serverless AWS Fargate, moving secrets to HashiCorp Vault to be fetched
```

Comment:
Make clear the business benefit - why containerize? Why pick Fargate as the deployment instead of whatever legacy platform this was deployed on?

Reply:
I think the issue with that is this bullet becomes pretty long. I'll look at how much room I have to play with but that's my concern with adding business impact onto this AWS/tech focused bullet

Reply:
Resolved in a call with you